ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/ExecutionStepFactory.java

```diff
@@ -149,7 +150,7 @@ public static TableSource tableSource(
         formats,
         timestampColumn,
         sourceSchema,
-        SystemColumns.CURRENT_PSEUDOCOLUMN_VERSION_NUMBER,
+        pseudoColumnVersion,
```

Comment:
Ignore this change, included in this commit due to my lack of proficiency in Git.